tests/native_api_tests/test_api_model.cpp

```diff
@@ -878,10 +900,20 @@ TEST_CASE_TEMPLATE(
 
     Options const batch_options{};
     Model model{50.0, input_dataset};
-    if constexpr (std::is_same_v<id_check_type, invalid_id_t>) {
-        CHECK_THROWS_AS(model.calculate(batch_options, batch_output_dataset, update_dataset), PowerGridBatchError);
-    } else {
-        model.calculate(batch_options, batch_output_dataset, update_dataset);
+
+    SUBCASE("Single update") {
+        if constexpr (std::is_same_v<id_check_type, invalid_id_t>) {
+            CHECK_THROWS_AS(model.update(update_dataset), PowerGridError);
+        } else {
+            model.update(update_dataset);
```

Comment:
Nitpick:
```suggestion
            CHECK_NOTHROW(model.update(update_dataset));
```

Same suggestion for the "Batch update" subcase below.